src/store/annotations/reducer.ts

```diff
@@ -1,21 +1,31 @@
 import { createReducer, combineReducers } from '@reduxjs/toolkit';
 import { AnnotationsState } from './types';
-import { createAnnotationAction, setActiveAnnotationIdAction } from './actions';
+import { createAnnotationAction, fetchAnnotationsAction, setActiveAnnotationIdAction } from './actions';
 
 const activeAnnotationId = createReducer<AnnotationsState['activeId']>(null, builder =>
     builder.addCase(setActiveAnnotationIdAction, (state, { payload: annotationId }) => annotationId),
 );
 
 const annotationsAllIds = createReducer<AnnotationsState['allIds']>([], builder =>
-    builder.addCase(createAnnotationAction.fulfilled, (state, { payload: { id } }) => {
-        state.push(id);
-    }),
+    builder
+        .addCase(createAnnotationAction.fulfilled, (state, { payload: { id } }) => {
+            state.push(id);
+        })
+        .addCase(fetchAnnotationsAction.fulfilled, (state, { payload }) => {
+            state.concat(payload.map(({ id }) => id));
```

Comment:
Assuming that the box-annotations store only contains the annotations for the given version, should this be a `concat` or a replacement of what's in state?

Reply:
I would have to keep track of and cancel pending requests if we take that approach, to avoid responses from overwriting each other. Since the annotation ids are unique, I don't see an immediate issue with them living in the store. Thoughts?